src/main/python/cfn_sphere/stack_configuration/__init__.py

```diff
@@ -9,22 +9,20 @@
 class Config(object):
     def __init__(self, config_file=None, config_dict=None, cli_params=None):
 
-        if config_dict:
-            self.dict = config_dict
+        if isinstance(config_dict, dict):
```

Comment:
Let me see, if I get this right, the Config object can either hold a config from a dict OR a config from a file, but not both?